src/test/java/Homework23_Serge.java

```diff
@@ -0,0 +1,25 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class Homework23_Serge extends BaseTest {
```

Comment:
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example `public class HomeWork23`